### ISSUE TITLE
making plugin transforms as peer dependency

### ISSUE THREAD

**Issue by jung-kim** [ariporad/babel-preset-features#2]
I think it would make sense to make all the below babel plugins as peer dependencies instead of dev dependencies to reduce duplicates being pulled in via npm

``````
    "babel-plugin-transform-async-to-generator": "^6.3.13",
    "babel-plugin-transform-es2015-arrow-functions": "^6.3.13",
    "babel-plugin-transform-es2015-block-scoped-functions": "^6.3.13",
    "babel-plugin-transform-es2015-classes": "^6.3.15",
    "babel-plugin-transform-es2015-computed-properties": "^6.3.13",
    "babel-plugin-transform-es2015-constants": "^6.1.4",
    "babel-plugin-transform-es2015-destructuring": "^6.3.15",
    "babel-plugin-transform-es2015-for-of": "^6.3.13",
    "babel-plugin-transform-es2015-function-name": "^6.3.21",
    "babel-plugin-transform-es2015-literals": "^6.3.13",
    "babel-plugin-transform-es2015-modules-commonjs": "^6.3.16",
    "babel-plugin-transform-es2015-object-super": "^6.3.13",
    "babel-plugin-transform-es2015-parameters": "^6.3.26",
    "babel-plugin-transform-es2015-shorthand-properties": "^6.3.13",
    "babel-plugin-transform-es2015-spread": "^6.3.14",
    "babel-plugin-transform-es2015-sticky-regex": "^6.3.13",
    "babel-plugin-transform-es2015-template-literals": "^6.3.13",
    "babel-plugin-transform-es2015-typeof-symbol": "^6.3.13",
    "babel-plugin-transform-es2015-unicode-regex": "^6.3.13",
    "babel-plugin-transform-es3-member-expression-literals": "^6.3.13",
    "babel-plugin-transform-es3-property-literals": "^6.3.13",
    "babel-plugin-transform-es5-property-mutators": "^6.3.13",
    "babel-plugin-transform-jscript": "^6.3.13",
    "babel-plugin-transform-regenerator": "^6.3.26"```
``````

**Comment by ariporad** [comment]
@codingtwinky: As of right now, they're just dependencies. But I agree, we should just depend on them directly, instead of via presets.

**Comment by ariporad** [comment]
This has been fixed in v2.0.5 (I think). Please re-open if it's still a problem.

**Comment by ariporad** [comment]
This has been fixed in v2.0.5 (I think). Please re-open if it's still a problem.
